develop/toc.yml

```diff
@@ -5105,14 +5105,21 @@
   items:
   - name: Platform-independent CI/CD
     topicUid: Platform_independent_CICD
+  - name: CI/CD using command line
+    topicUid: CICD_Command_Line_Examples
+    items:
+    - name: Command line examples
+      topicUid: CICD_Command_Line_Examples
   - name: CI/CD using GitHub
     topicUid: CICD_on_GitHub
     items:
+    - name: GitHub example workflows
+      topicUid: CICD_GitHub_Examples
     - name: GitHub starter workflows
       topicUid: github_starter_workflows
     - name: GitHub validation requirements
       topicUid: github_validation_requirements
-    - name: Deploying from GitHub
+    - name: MarketPlace action
```

Comment:
I'm not sure I completely agree. The missing information is likely a more detailed "how-to" very specific to the CICD technology you're wanting to use. All of which is a simple 'google' away or will likely already be known for any cicd engineer wanting to make such a set-up. Concourse, Azure Devops, GitHub Actions, GitLab, ...  all of those have their own help & documentation that tells you what to push until you get to your "empty script". At that point you can copy paste one of the examples we prove and add to it. (or add our example to an existing script)

I'd like to avoid adding too much extra industry-standard information and focus more on the DataMiner specific information you need to immediately get things up and running in your technology of choice.

Would it be good enough you think, if I just add some links to the help pages of those technologies?

---------
There is a command line in Windows and a command line in Linux (ubuntu).  Windows usually goes with Powershell these days and Ubuntu has the Terminal with bash scripts. So that page shows both options, you can pick which one based on your Operating System. It doesn't make much sense to see that as consecutive steps as they are for different operating systems.

I can add an extra line or two, mentioning that there is an example in powershell for windows users and an example with ubuntu terminal for ubuntu users wishing to use the command line.

---------

I was afraid of changing the file names and UID's of the existing "Deploying from Github" because I think we have linked to that several times in answers, e-mails and dojo posts. However that name doesn't make sense any more, it was too generic and it makes it confusing for the users.  The page is talking about the Github marketplace action that allows deployment. But for most users we would refer them now to use the platform-independent .NET Tools we just released.

If you think it's OK, I can look into adjusting the filename & UID of the old page then and not just the treecontrol displayname and internal title.